internal/tiltfile/tiltfile.go

```diff
@@ -454,17 +454,18 @@ func handleSkylarkErr(thread *skylark.Thread, err error) error {
 	return err
 }
 
-// GetManifestConfigsAndGlobalYAML executes the Tiltfile to create manifests for all resources and
-// a manifest representing the global yaml.
-func (t Tiltfile) GetManifestConfigsAndGlobalYAML(ctx context.Context, names ...model.ManifestName) ([]model.Manifest, model.YAMLManifest, error) {
+// GetAllManifests executes the Tiltfile to create manifests for all resources and
+// a manifest representing the global yaml. NOTE: If you try to call this on only
```

Comment:
hmmm...this seems like a fundamental design flaw in the API.

Can we change LoadTiltfile to take all the manifest names? 

fundamentally, it seems like global_yaml depends on the manifest names passed in, which implies that the manifest names passed in should be a global property of the Tiltfile.